EnglishFilter/sections/whitelist_stealth.txt

```diff
@@ -26,6 +26,8 @@
 !
 ! https://github.com/AdguardTeam/AdguardFilters/issues/36048 [ Stealth Mode - Block third-party Authorization header]
 @@||api.pinger.com$stealth
+! https://github.com/AdguardTeam/AdguardFilters/issues/36278 [Stealth Mode - Referrer]
+@@||yaeby.me^$stealth

```

Comment:
If this is supposed to fix "html5: Video file not found" on video, not working.

This rule works for me (also referrer):
@@||rykoeb.com/remote_control.php^$stealth